From: DaPigGuy <mcpepig123@gmail.com>
Date: Thu, 19 Mar 2026 18:48:13 -0700
Subject: shell: Fix overcounting of misc. symbols for password strength

There was previously a branch for lowercase symbols that was removed.
Lowercase symbols should not count as miscellaneous symbols, which
increase the password strength.

Fixes ecb7dcc5d67c33d2d5155ecfc32fc36a7e4cfa08

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/4143>

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/b8dbf86
---
 src/shell-keyring-prompt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/shell-keyring-prompt.c b/src/shell-keyring-prompt.c
index 821505b..28c23d7 100644
--- a/src/shell-keyring-prompt.c
+++ b/src/shell-keyring-prompt.c
@@ -628,7 +628,7 @@ calculate_password_strength (const gchar *password)
         digit++;
       else if (g_ascii_isupper (password[i]))
         upper++;
-      else
+      else if (!g_ascii_islower (password[i]))
         misc++;
     }
 
